spec/models/elastic_indexed_document_spec.rb

```diff
@@ -248,14 +248,18 @@
             expect(ElasticIndexedDocument.search_for(q: 'dolphn*', affiliate_id: affiliate.id, language: affiliate.indexing_locale).total).to eq(0)
             expect(ElasticIndexedDocument.search_for(q: 'tx?', affiliate_id: affiliate.id, language: affiliate.indexing_locale).total).to eq(0)
             expect(ElasticIndexedDocument.search_for(q: 'dolph*', affiliate_id: affiliate.id, language: affiliate.indexing_locale).total).to eq(1)
-            expect(ElasticIndexedDocument.search_for(q: 'dolph?n', affiliate_id: affiliate.id, language: affiliate.indexing_locale).total).to eq(1)
+            expect(ElasticIndexedDocument.search_for(q: 'dolph?ns', affiliate_id: affiliate.id, language: affiliate.indexing_locale).total).to eq(1)
```

Comment:
@peggles2, we may as well delete this line, as this is now testing the same thing as `expect(ElasticIndexedDocument.search_for(q: 't?xt'...`